rts/at/integration/src/main/java/org/jboss/narayana/rest/integration/RecoveryManager.java

```diff
@@ -125,7 +123,7 @@ private OutputObjectState getParticipantInformationOutputState(final Participant
         return new byte[] {};
```

Comment:
This method should be throwing an exception to let the developer know that the implementation of Participant should be serializable or implement PersistableParticipant.